### PR TITLE
Field: add `SetInt64` and support for `uint8` `uint16` ... etc in `SetInterface`

### PR DESCRIPTION
**Pull request by gbotrel**

### CODE REVIEW

**Review by Tabaie — APPROVED** [review]
LGTM, there's just that one redundant import

File: field/internal/templates/element/tests.go [hunk 1]
```diff
@@ -7,12 +7,14 @@ import (
 	"encoding/json"
 	"math/big"
 	"math/bits"
+	"fmt"
 	{{if .NoCarry}} mrand "math/rand" {{end}}
 	"testing"
 	{{if .UseAddChain}}	"fmt" {{ end }}
```
[the remark this answers is not in the document]
Comment (gbotrel) [reply field/internal/templates/element/tests.go:13]: nope it is used in the added tests 👍

[the remark this answers is not in the document]
Comment (Tabaie) [reply field/internal/templates/element/tests.go:13]: I'm referring to line 13, which conditionally imports "fmt"

[the remark this answers is not in the document]
Comment (gbotrel) [reply field/internal/templates/element/tests.go:13]: Haaa got you my bad :-)

**Pull request merged**